big_tests/tests/instrument_cets_SUITE.erl

```diff
@@ -0,0 +1,46 @@
+-module(instrument_cets_SUITE).
+-include_lib("eunit/include/eunit.hrl").
+
+-compile([export_all, nowarn_export_all]).
+
+-import(distributed_helper, [mim/0, mim2/0, rpc/4]).
+-import(domain_helper, [host_type/1]).
+-import(mongooseimctl_helper, [rpc_call/3]).
+
+all() ->
+    [{group, instrument}].
+
+groups() ->
+    [{instrument, [], cases()}].
+
+cases() ->
+    [check_instrumentation].
+
+init_per_suite(Config) ->
+    case rpc_call(mongoose_config, lookup_opt, [[internal_databases, cets]]) of
+        {error, not_found} ->
+            {skip, "CETS is not configured"};
+        {ok, _} ->
+            instrument_helper:start(instrument_helper:declared_events(mongoose_metrics_probe_cets)),
```

Comment:
This is wrong, because `instrument_helper:declared_events/1` is meant only for `mod_*` modules, which have `#{host_type => HostType}` labels, while you have empty labels. The first problem is that your `instrumentation` function succeeds for `#{host_type => HostType}`, while it should not. The second problem is that you just get a `{cets_info, #{host_type => ...}}` event instead of `{cets_info, #{}}`. As a result, you are listening for the wrong event. So either do as I suggest below, or extend the helper to work without host types as well.

```suggestion
            instrument_helper:start([{cets_info, #{}}]),
```